ui/component/searchTopClaim/view.jsx

```diff
@@ -51,33 +61,61 @@ export default function SearchTopClaim(props: Props) {
     }
   }, [doResolveUris, uriFromQuery, channelUriFromQuery]);
 
-  if (!winningUri) {
-    return null;
-  }
-
   return (
     <section className="search">
       <header className="search__header">
-        <div className="claim-preview__actions--header">
-          <span className="media__uri">
-            {__('Most supported')}
-            <HelpLink href="https://lbry.com/faq/tipping" />
-          </span>
-        </div>
-        <div className="card">
-          <ClaimPreview
-            hideRepostLabel
-            uri={winningUri}
-            type="large"
-            placeholder="publish"
-            properties={claim => (
-              <span className="claim-preview__custom-properties">
-                <ClaimEffectiveAmount uri={winningUri} />
-              </span>
-            )}
-          />
-        </div>
-        {!hideLink && (
+        {winningUri && (
+          <div className="claim-preview__actions--header">
+            <a
+              className="help"
+              href="https://lbry.com/faq/trending"
+              title={__('Learn more about LBRY Credits on %DOMAIN%', { DOMAIN })}
+            >
+              <I18nMessage
+                tokens={{
+                  lbc: <LbcSymbol />,
+                }}
+              >
+                Most supported %lbc%
+              </I18nMessage>
+            </a>
+          </div>
+        )}
+        {winningUri && (
+          <div className="card">
+            <ClaimPreview
+              hideRepostLabel
+              uri={winningUri}
+              type="large"
+              placeholder="publish"
+              properties={claim => (
+                <span className="claim-preview__custom-properties">
+                  <ClaimEffectiveAmount uri={winningUri} />
+                </span>
+              )}
+            />
+          </div>
+        )}
+        {!winningUri && uriFromQuery && (
+          <div className={'empty empty--centered'}>
```

Comment:
Please use `className="empty empty--centered"` if we aren't using `classnames` or a js variable in the classname